compiler/src/iree/compiler/Dialect/Flow/Transforms/test/verify_no_f64.mlir

```diff
@@ -0,0 +1,21 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(func.func(iree-verify-input-legality))" --verify-diagnostics %s -split-input-file
```

Comment:
for easier discoverability we try to keep the test files relatively 1:1 with the passes, so these could go in verify_input_ir.mlir with the other tests related to the pass

Reply:
done